providers/implementations/include/prov/seeding.h

```diff
@@ -10,6 +10,38 @@
 #include "prov/provider_ctx.h"
 #include "crypto/rand_pool.h"
 
+/* Entropy quality for different operating systems */
```

Comment:
I would add a bit more commentary, to remind us where these numbers come from.  Perhaps something along the lines of this:

The numbers given here must match the number given to `ossl_rand_pool_bytes_needed()` in the platform specific seeding implementations in providers/implementations/rand/seeding,
and in case a platform has several possible implementations, the largest number given
there.